feature_extractor.py

```diff
@@ -132,7 +132,7 @@ def read_features(file_path, cache=None):
     features = None
     with open(file_path, 'r') as fp:
         data = fp.read().splitlines(keepends=False)
-        features = np.zeros((len(data), 4096))
+        features = np.zeros((len(data), 2048))
```

Comment:
Should check the feature extractor type used in order to specify the right dimensions

Reply:
I think so too. I will fix it.